lib/cli.js

```diff
@@ -270,6 +271,16 @@ exports.run = async () => { // eslint-disable-line complexity
 
 	const files = cli.input.map(file => path.relative(resolveTestsFrom, path.resolve(process.cwd(), file)));
 
+	for (const file of files) {
+		if (!fs.existsSync(file)) {
```

Comment:
Using `statSync()` (see below) should be sufficient. It'll throw if the file (or its path) does not exist, but you can catch the error and use `error.code === 'ENOENT' || error.code === 'ENOTDIR'` to recognize whether the file existed. See https://nodejs.org/api/errors.html#errors_common_system_errors.

Reply:
No need for `error.code === 'ENOTDIR'`